<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestAdvanceSetting</class>
 <widget class="QDialog" name="TestAdvanceSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高级设置</string>
  </property>
  <widget class="QLineEdit" name="msgLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入自定义消息</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reliable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可靠发送</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="customRender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义渲染：</string>
   </property>
  </widget>
  <widget class="QWidget" name="dialogLocalView" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cmdId">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>消息ID：1 - 10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sendCustomCmdMsg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义发送</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ordered">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>有序</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义消息：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="otherRoomNameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>请输入跨房房间号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入跨房房间号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="otherUserNameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>请输入跨房用户ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入跨房用户ID：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="connectOtherRoomBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始 / 结束跨房连麦</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeatCount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送数据次数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="res">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sendSEIMsg">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>嵌入视频帧发送</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>271</y>
     <width>371</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <zorder>listView_3</zorder>
  <zorder>listView_2</zorder>
  <zorder>listView</zorder>
  <zorder>dialogLocalView</zorder>
  <zorder>msgLineEdit</zorder>
  <zorder>reliable</zorder>
  <zorder>customRender</zorder>
  <zorder>cmdId</zorder>
  <zorder>label_2</zorder>
  <zorder>sendCustomCmdMsg</zorder>
  <zorder>ordered</zorder>
  <zorder>label</zorder>
  <zorder>otherRoomNameLineEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>otherUserNameLineEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>connectOtherRoomBox</zorder>
  <zorder>repeatCount</zorder>
  <zorder>label_3</zorder>
  <zorder>res</zorder>
  <zorder>sendSEIMsg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
